stellar-contract-env-host/src/test/examples.rs

```diff
@@ -0,0 +1 @@
+
```

Comment:
👀 What's this do?

Reply:
Oh, that's an edit artifact left over from when I was trying to decide what to name the submodule. Snuck in, will remove.